src/unnamed-engine/src/core/scripting/DotnetHost.cpp

```diff
@@ -12,7 +12,7 @@ constexpr std::string_view RUNTIME_CONFIG_JSON("assembly-test.runtimeconfig.json
 #if WIN32
 #define HOST_FXR_PATH "host/fxr/8.0.0/hostfxr.dll"
 #elif __linux__
-#define HOST_FXR_PATH "host/fxr/8.0.0/libhostfxr.so"
+#define HOST_FXR_PATH "host/fxr/8.0.2/libhostfxr.so"
```

Comment:
Are we updating our .NET support? I think it might be worth checking out handling minor versions automatically c:

Reply:
I created issue #22 to handle this case here, you can mark it as resolved

Reply:
Yeah, it was because I had a minor version. Sorry, GH did not notify me. :c